test/integrations/destinations/intercom/router/data.ts

```diff
@@ -335,7 +378,51 @@ export const data = [
                   jobId: 3,
                 },
               ],
-              statusCode: 200,
+              statusCode: 299,
+            },
+            {
+              batched: false,
+              batchedRequest: {
+                body: {
+                  JSON: {
+                    id: '657264e9018c0a647s45',
+                  },
+                  XML: {},
+                  FORM: {},
+                  JSON_ARRAY: {},
+                },
+                endpoint: 'https://api.eu.intercom.io/contacts/70701240741e45d040/companies',
+                files: {},
+                headers: {
+                  Authorization: 'Bearer testApiKey',
```

Comment:
Please update them to new format of test cases.Take Klaviyo test cases as reference